### ISSUE TITLE
NTR: Nails: lunulae enlarged (Extremities, joints & nails)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7974]
New term request Nails: lunulae enlarged (elthat_150801155731): Lunula; crescent-shaped whitish area of the bed of a fingernail or toenail  UMLS: 0423831

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000137
- label: Enlarged lunula
- parents: [Abnormality of the nail (HP:0001597)](https://hpo.jax.org/app/browse/term/HP:0001597)
- pmid: PMID:30245952
- def: Abnormal enlargement of the visible portion of the distal nail matrix that extends beyond the proximal nail fold, which is refered to as the lunula.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7974
- synonyms: Macrolunula(NONE)|Nails: lunulae enlarged(NONE)
